tests/parsers/test.manifestjson.js

```diff
@@ -387,8 +387,11 @@ describe('ManifestJSONParser', () => {
         'worker-src web.example.com:80',
         'worker-src web.example.com:443',
 
+        // flag unsafe-eval
+        "script-src 'self' 'unsafe-eval';",
+
         // Properly match mixed with other directives
-        "script-src https: 'unsafe-eval'; object-src 'self'",
+        "script-src https: 'unsafe-inline'; object-src 'self'",
```

Comment:
This is throwing a warning because of the `https` in there which means the dev tries to enable remote script execution unless we're talking about different things?

Reply:
Ah yes, right. The change from `unsafe-eval` to `unsafe-inline` confused me.